docs/04-docker-setup-postgres-connect-with-r.md

```diff
@@ -37,24 +37,31 @@ system2("docker", "version", stdout = TRUE, stderr = TRUE)
 ## [18] "  Experimental:     true"
 ```
 
-The convention we use in this book is to assemble a command with `paste0` so that the parts of the command can be specified separately.
+The convention we use in this book is to assemble a command with `paste0` so that the parts of the command can be specified separately.  This chunk just constructs the command, but does not execute.  If you have problems, you can copy the command and execute in your terminal session.
```

Comment:
Should this say `glue`, rather than `paste0`? The next chunk of code uses `glue` to construct the `docker_cmd`.

Reply:
I think I've swapped all of the paste0() commands for glue().